app/models/manageiq/providers/azure_stack/inventory/collector/cloud_manager.rb

```diff
@@ -1,4 +1,7 @@
 class ManageIQ::Providers::AzureStack::Inventory::Collector::CloudManager < ManageIQ::Providers::AzureStack::Inventory::Collector
```

Comment:
Please add a comment at the top of the file that this is a reference implementation and that it bases on the lates profile currently available, the 2018